UMAProject/Assets/UMA/Editor/AssetTreeElement.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using UnityEngine;
+using Random = UnityEngine.Random;
```

Comment:
Random shouldn't be used here and doesn't seem to be referenced. I wonder if this is how suggestions for deleting lines should be. 
```suggestion
```